test/e2e/specs/editor/blocks/navigation.spec.js

```diff
@@ -930,6 +930,467 @@ test.describe( 'Navigation block', () => {
 			await expect( linkControl.getSearchInput() ).not.toBeVisible();
 		} );
 	} );
+
+	test.describe( 'Navigation colors', () => {
+		test.beforeAll( async ( { requestUtils } ) => {
+			//we want emptytheme because it doesn't have any styles
+			await requestUtils.activateTheme( 'emptytheme' );
+			await requestUtils.deleteAllTemplates( 'wp_template_part' );
+			await requestUtils.deleteAllTemplates( 'wp_template' );
+			await requestUtils.deleteAllPages();
+			await requestUtils.deleteAllMenus();
+		} );
+
+		test.beforeEach( async ( { admin, editor, requestUtils } ) => {
+			await admin.visitSiteEditor( {
+				postId: 'emptytheme//header',
+				postType: 'wp_template_part',
+			} );
+			await editor.canvas.click( 'body' );
+			await requestUtils.createNavigationMenu( {
+				title: 'Hidden menu',
+				content:
+					'<!-- wp:navigation-submenu {"label":"First link","type":"custom","url":"http://www.wordpress.org/","kind":"custom"} --><!-- wp:navigation-link {"label":"Second Link","type":"custom","url":"http://www.wordpress.org/","kind":"custom"} /--><!-- /wp:navigation-submenu -->',
+				attributes: { openSubmenusOnClick: true },
+			} );
+			await editor.insertBlock( {
+				name: 'core/navigation',
+			} );
```

Comment:
You could probably speed things up by getting the ID from the creation and passing as `ref` to the block.

Reply:
ah, smart!